app/src/test/java/com/android/voyageur/model/trip/TripsViewModelTest.kt

```diff
@@ -49,8 +49,9 @@ class TripsViewModelTest {
 
   @Test
   fun getTripsCallsRepository() {
-    tripsViewModel.getTrips()
-    verify(tripsRepository).getTrips(any(), any())
+
+    // Simulate a successful result from the repository
```

Comment:
I will redo this part when we can mock the firebase auth

Reply:
yes, good idea but do not forget to do so please :)